dashboard/lib/pd/certificate_renderer.rb

```diff
@@ -0,0 +1,124 @@
+module Pd
+  class CertificateRenderer
+    HARDCODED_CSD_FACILITATOR = 'Dani McAvoy'
+    HARDCODED_CSP_FACILITATOR = 'Brook Osborne'
+    HARDCODED_OTHER_TEACHERCON_FACILITATOR = 'Code.org team'
+
+    # Given a PD enrollment, renders a workshop completion certificate.
+    #
+    # NOTE:
```

Comment:
nit: use [@note](https://www.rubydoc.info/gems/yard/file/docs/Tags.md#note)

Reply:
TIL!  Also :joy::
```rb
# @note This method should only be used in outer space.
def eject; end
```

Reply:
lol I did not read what I link to